src/Catalyst.Core.Modules.Dfs.Tests/IntegrationTests/CoreApi/ObjectApiTest.cs

```diff
@@ -0,0 +1,224 @@
+        }
+
+        [Fact]
+        public async Task Put_Get_Data()
+        {
+            var adata = Encoding.UTF8.GetBytes("alpha");
+            var bdata = Encoding.UTF8.GetBytes("beta");
+            var alpha = new DagNode(adata);
+            var beta = await ipfs.ObjectApi.PutAsync(bdata, new[] {alpha.ToLink()});
+            var node = await ipfs.ObjectApi.GetAsync(beta.Id);
+            Assert.Equal(beta.DataBytes, node.DataBytes);
+            Assert.Equal(beta.Links.Count(), node.Links.Count());
+            Assert.Equal(beta.Links.First().Id, node.Links.First().Id);
+            Assert.Equal(beta.Links.First().Name, node.Links.First().Name);
+            Assert.Equal(beta.Links.First().Size, node.Links.First().Size);
+        }
+
+        [Fact]
+        public async Task Data()
+        {
+            var adata = Encoding.UTF8.GetBytes("alpha");
+            var node = await ipfs.ObjectApi.PutAsync(adata);
+            using (var stream = await ipfs.ObjectApi.DataAsync(node.Id))
+            {
+                var bdata = new byte[adata.Length];
+                stream.Read(bdata, 0, bdata.Length);
+                Assert.Equal(adata, bdata);
+            }
+        }
+
+        [Fact]
+        public async Task Links()
+        {
+            var adata = Encoding.UTF8.GetBytes("alpha");
+            var bdata = Encoding.UTF8.GetBytes("beta");
+            var alpha = new DagNode(adata);
+            var beta = await ipfs.ObjectApi.PutAsync(bdata, new[] {alpha.ToLink()});
+            var links = await ipfs.ObjectApi.LinksAsync(beta.Id);
+            Assert.Equal(beta.Links.Count(), links.Count());
+            Assert.Equal(beta.Links.First().Id, links.First().Id);
+            Assert.Equal(beta.Links.First().Name, links.First().Name);
+            Assert.Equal(beta.Links.First().Size, links.First().Size);
+        }
+
+        [Fact]
+        public async Task Stat()
+        {
+            var data1 = Encoding.UTF8.GetBytes("Some data 1");
+            var data2 = Encoding.UTF8.GetBytes("Some data 2");
+            var node2 = new DagNode(data2);
+            var node1 = await ipfs.ObjectApi.PutAsync(data1,
+                new[] {node2.ToLink("some-link")});
+            var info = await ipfs.ObjectApi.StatAsync(node1.Id);
+            Assert.Equal(1, info.LinkCount);
+            Assert.Equal(11, info.DataSize);
+            Assert.Equal(64, info.BlockSize);
+            Assert.Equal(53, info.LinkSize);
+            Assert.Equal(77, info.CumulativeSize);
+        }
+
+        [Fact]
+        public async Task Get_Nonexistent()
+        {
+            var data = Encoding.UTF8.GetBytes("Some data for net-ipfs-engine-test that cannot be found");
+            var node = new DagNode(data);
+            var id = node.Id;
+            var cs = new CancellationTokenSource(500);
+            try
+            {
+                var _ = await ipfs.ObjectApi.GetAsync(id, cs.Token);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused local variable '_'.](https://app.codacy.com/manual/catalyst-network/Catalyst.Framework/pullRequest?prid=4832783)